src/main/java/game/domain/RacingGameCreateService.java

```diff
@@ -0,0 +1,17 @@
+package game.domain;
+
+import java.util.List;
+
+public class RacingGameCreateService {
+
+    private RacingGameCreateService(){}
+
+    public static RacingGameCreateService create() {
+        return new RacingGameCreateService();
+    }
+
+    public List<String> getRacingNames(final String carNames) {
```

Comment:
```suggestion
    public List<String> getCarNames(final String carNames) {
```
가 맞지 않을까요? 밑에서 쓰는 부분에서도 carNames로 받고 있길래 코멘트 달아봅니다

Reply:
넵 그 부분이 맞습니다.. :)